.versionpress/schema.yml

```diff
@@ -0,0 +1,15 @@
+pods_relationship:
+  table: podsrel
+  id: id
+  references:
+    pod_id: post
+    field_id: post
+    item_id: post
```

Comment:
The complexity of what Pods does makes this a little trickier, the item ID could be related to a post, a term, a user, etc depending on the pod the field was added on.